src/Aspire.ProjectTemplates/templates/aspire-starter/9.4/Aspire-StarterApplication.1.ApiService/Aspire-StarterApplication.1.ApiService.http

```diff
@@ -0,0 +1,6 @@
+@ApiService_HostAddress = http://localhost:5301
```

Comment:
Must be missing something, but shouldn't this be the https one? So port 7301?

Reply:
No, this matches what the API template does, and it's because if the app is launched with the "http" launch profile, there won't be an HTTPS port, only HTTP. The HTTP port is consistent between both the "http" and "https" launch profiles so we use it here.